store/storetest/emoji_store.go

```diff
@@ -30,33 +30,26 @@ func testEmojiSaveDelete(t *testing.T, ss store.Store) {
 		Name:      model.NewId(),
 	}
 
-	if _, err := ss.Emoji().Save(emoji1); err != nil {
-		t.Fatal(err)
-	}
+	_, err := ss.Emoji().Save(emoji1)
+	require.Nil(t, err, err)
 
-	if len(emoji1.Id) != 26 {
-		t.Fatal("should've set id for emoji")
-	}
+	require.Equal(t, len(emoji1.Id), 26, "should've set id for emoji")
```

Comment:
Please use `assert` for this because the test can continue if this fails. Also you can use [`assert.Len`](https://godoc.org/github.com/stretchr/testify/assert#Len) for this.

Reply:
Arent we replacing fatal with require? I thought because we are using t.fatal here, we want to not continue?

Reply:
You are correct. But this campaign gives us the chance to improve the code a bit and hence we want to change the behaviour.